types/rlp_encoding_test.go

```diff
@@ -281,53 +296,59 @@ func TestRLPMarshall_And_Unmarshall_TxType(t *testing.T) {
 func testRLPData(arena *fastrlp.Arena, ommitValues map[string]bool) []byte {
 	vv := arena.NewArray()
 
-	_, ommit := ommitValues["Nonce"]
-	if !ommit {
+	if ommit, _ := ommitValues["ChainID"]; !ommit {
```

Comment:
I hate to be the nitpicker here but it's `omit` and `omitted` 😄

Reply:
Fixed